sdk/canton/BUILD.bazel

```diff
@@ -1096,13 +1096,13 @@ proto_gen(
 
 daml_compile(
     name = "community_participant_admin-workflows",
-    srcs = glob(["community/participant/src/main/daml/AdminWorkflows/**/*.daml"]),
+    srcs = glob(["community/participant/src/main/daml/canton-builtin-admin-workflow-ping/**/*.daml"]),
     project_name = "AdminWorkflows",
 )
 
 daml_compile(
     name = "community_participant_party-replication",
-    srcs = glob(["community/participant/src/main/daml/PartyReplication/**/*.daml"]),
+    srcs = glob(["community/participant/src/main/daml/canton-builtin-admin-workflow-party-replication-alpha/**/*.daml"]),
     project_name = "PartyReplication",
 )
 
```

Comment:
The names of the dar's have also changed, so we likely also need:

- `s/AdminWorkflows.dar/canton-builtin-admin-workflow-ping.dar/`
- `s/PartyReplication.dar/canton-builtin-admin-workflow-party-replication-alpha.dar/`

in lines 1109-1119 below